kobo/hub/views.py

```diff
@@ -232,7 +231,7 @@ def task_log_json(request, id, log_name):
         next_poll = LOG_WATCHER_INTERVAL
 
     if six.PY3:
-        content = str(content, encoding=locale.getpreferredencoding())
+        content = str(content, encoding="utf-8")
```

Comment:
I think this is reasonable, but I also worry about what other bugs might be triggered in an environment where locale.getpreferredencoding() returns 'ascii'.  Not only in our code but also other libraries used by us, where enforcing utf8-only might be less acceptable.

It would be a good idea to find and fix whatever causes locale.getpreferredencoding() to return ascii rather than utf8 for the relevant service, otherwise it wouldn't be surprising for additional similar bugs to crop up repeatedly.

Reply:
I ran getpreferredencoding() in our Openshift pub-hub container and it returned utf-8. This makes me assume that this only happens intermittently and it may be caused by Django itself. I have actually discovered this error when I was debugging a different issue, so maybe this is just a consequence of another bug (OIDC related). Though I don't think I have the skills to disentangle this and pinpoint the root cause.